src/canvas/tileLayer.js

```diff
@@ -0,0 +1,87 @@
+geo.canvas.tileLayer = function () {
```

Comment:
@aashish24 This isn't an override, this is additive.  The way we do class inheritance in geojs doesn't really support multiple inheritance.  As such, for all of the tileLayers (d3, gl, and now canvas), we modify the instance of the tileLayer after it has been created.  This code is just doing the same thing as the d3 and gl renderers.  Ideally, we'd be using a more common javascript inheritance mechanism and it would be cleaner.